### ISSUE TITLE
Subcategories for profession caches

### ISSUE THREAD

**Issue by Vedarety** [teelolws/TLDRMissions#43]
TLDR version: More options for limiting missions by anima cost or rewards.

**Slightly longer version**
Most gold missions for example have a 10 anima base cost, but "Learn from the Master" costs 150 anima. Same with the "Joint Operations: The Hunt" reputation rewards mission, which costs 150 for 750 rep, compared to the "normal" reputation missions which cost 10 anima for 375 rep. The difference can hurt when trying to be anima-neutral with the tables, so being able to ignore these would be nice.

Another point, some crafting material caches are not worth it (subjectively) to run missions for, for example the Meat Caches. It would be nice to have a split of them in the options like: "Mining Caches", "Skinning Caches", "Herbalism Caches", "Fishing Caches", "Meat Caches", "Enchanting Caches" as an advanced option.

**Comment by teelolws** [comment]
For point 1, I could add a slider with the most amount of anima you're willing to spend on missions?

For point 2, this is something I'll consider in the future. I have other reward types I need to slot in before I add micromanaging the ones already there.

**Comment by Vedarety** [comment]
A slider/number input does seem like a smart solution.

**Comment by teelolws** [comment]
Coming in the next update later today:
![image](https://user-images.githubusercontent.com/49792789/164104237-2fc0e356-e9ad-4a79-b1b8-431f7bf1181d.png)

**Comment by Nillx** [comment]
> 
> 
> I have other reward types I need to slot in before I add micromanaging the ones already there.

Does that mean the epic items from missions?

**Comment by teelolws** [comment]
Gear, and Campaign Missions are two I want to add. 

I also want to change the CheckBoxes to support crosses - exclude rewards you don't want. So if a mission had two rewards, say, gold + anima, you "cross out" anima, but "tick" gold, the exclusion will apply first and it won't try that mission.

**Comment by teelolws** [comment]
Subcategories coming in next update: 
![image](https://user-images.githubusercontent.com/49792789/165656530-4e780551-04cf-4e98-b467-46d79ae6f96d.png)


Gear in the future. Buttons there just haven't finished making it yet.

**Comment by Nillx** [comment]
That looks great!
